site/app/views/admin/UsersView.php

```diff
@@ -295,6 +297,133 @@ public function userForm($reg_sections, $rot_sections, $action, $use_database=fa
         return $return;
     }
 
+    /**
+     * Creates the form box to be displayed when copying or downloading emails on the students pages
+     * @param User[] $students  
+     * @param array  $reg_sections associative array representing registration sections in the system
+     * @param bool   $use_database
+     * @return string
+     */
```

Comment:
this function consist of html code for copy/download form that will be rendered for student tab